src/main/java/org/opensrp/repository/postgres/ClientsRepositoryImpl.java

```diff
@@ -661,6 +663,13 @@ public void removeAll() {
 	public List<Patient> findClientByRelationship(String relationship, String id) {
 		return convertToFHIR(findByRelationshipId(relationship, id));
 	}
+
+	@Override
+	public List<Client> findByClientTypeAndLocationId(String clientType, String locationId) {
```

Comment:
change `findFamilyMemberyByJurisdiction` to use this method

Reply:
change `findFamilyMemberByResidence`  this clause ` .andLastNameNotEqualTo("Family");` to ` .andClientTypeNotEqualTo("Family");`

Reply:
Done

Reply:
Done